src/System.Net.NameResolution/src/System/Net/DNS.cs

```diff
@@ -75,6 +94,38 @@ internal static IPHostEntry InternalGetHostByName(string hostName, bool includeI
             return ipHostEntry;
         } // GetHostByName
 
+        [Obsolete("GetHostByAddress is obsoleted for this type, please use GetHostEntry instead. http://go.microsoft.com/fwlink/?linkid=14202")]
+        public static IPHostEntry GetHostByAddress(string address)
+        {
+            if (NetEventSource.Log.IsEnabled()) NetEventSource.Enter(NetEventSource.ComponentType.Socket, "DNS", "GetHostByAddress", address);
+
+            if (address == null)
+            {
+                throw new ArgumentNullException("address");
+            }
+
+            GlobalLog.Print("Dns.GetHostByAddress: " + address);
```

Comment:
It looks like GlobalLog.Print is not [Conditional("Debug")].  As a result, this string concatenation is going to happen all the time.  I expect we want to guard it behind GlobalLog.IsEnabled.  Same for any other occurrences.